source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -3,6 +3,121 @@ title: Migrate from our old frameworks
 weight: 210
 ---
 
-# Migrate from our old frameworks
+# Migrate to GOV.UK Frontend from our old frameworks
 
-Read about how to [update from GOV.UK Elements and Frontend Toolkit](https://test_moving_docs--govuk-design-system-preview.netlify.com/get-started/updating-your-code/).
+You should migrate to GOV.UK Frontend if you're using one of the following old ('legacy') frameworks:
+
+- GOV.UK Frontend Toolkit
+- GOV.UK Template
+- GOV.UK Elements
+
+We're no longer updating our old frameworks, and they do not meet the [Web Content Accessibility Guidelines (WCAG 2.1) accessibility standard](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).
+
+After you include GOV.UK Frontend in your project, your service should look and behave the same as before. This means you can migrate one component at a time in your live service.
+
+Your service may be slightly slower while you migrate, because your users will need to download more CSS and JavaScript.
+
+## Audit your service's components
+
+To help you migrate, you should work with a designer to make a list of the components in your service, and compare it to [the components in GOV.UK Frontend](https://design-system.service.gov.uk/components/).
+
+You can use your list to see where you can replace similar elements in your service with one GOV.UK Frontend component. For example, if you have a range of similar buttons, you may be able to replace them all with the [button](https://design-system.service.gov.uk/components/button/) component.
+
+We recommend using a template engine like [Nunjucks](https://mozilla.github.io/nunjucks/) to abstract each component into a reusable macro (also called a 'partial' or 'template'). You can do this before or during your migration.
```

Comment:
We've already mentioned looking at the component pages, so pointing people to particular aspect of the component page feels like it would be 'content about content', which we usually try to avoid. I appreciate we know that the macro options are easy to miss, but until that's fixed we should probably avoid compensating by pointing users to them?

Reply:
I'd be really keen to point users to the table options now as it'll make the process of creating macros more straightforward, regardless of whether there's a problem we haven't solved. But I leave it up to you.

Reply:
Ok - I've given this a go in the next commit.